### DISCUSSION TITLE
Is it possible to replace (reverse engineering) the cs2 engine with csgo for best performance preserving access to cs2 official servers?

### DISCUSSION THREAD

**Discussion by sudg6** [danielkrupinski/Osiris#4338] [Q&A]
The title says it all.

**Comment by gr1ndy** [comment]
????

both engines run on different architecture? how do u want to achieve that LOL

**Comment by danielkrupinski** [comment]
Yes, but it would be easier to emulate communication with the server and write your own game client.

**Reply by sudg6** [reply]
Could you write brief logical steps to reach out to this scenario?
I think that Valve managers do not allow such a thing to happen!
